compiler/src/help.ml

```diff
@@ -17,7 +17,7 @@ let show_intrinsics asmOp fmt =
   let headers = [|
       "no size suffix";
       "one optional size suffix, e.g., “_64”";
-      "a zero/sign extend suffix, e.g., “_u32u16”";
+      "a zero/sign extend suffix, e.g., “_s16” or “_u32u16”";
```

Comment:
This is surprising: does it mean that it is now possible to write `MOVSX_s16`?

Reply:
Any updates on this issue?

Reply:
I don't know how to solve this.